02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,29 @@
+const {people} = require('../data');
+
+function getPeople(req,res){
+    res.status(200).json(people)
+}
+
+function addPerson(req,res){
+    const name = req.body.name;
```

Comment:
Looks good, but if you want something more modern, try restructuring the name from the object:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment